common.c

```diff
@@ -105,3 +114,76 @@ const Instr_t Factorial[PROGRAM_SIZE] = {
     Instr_Print,    // n
     Instr_Halt
 };
+
+cpu_t init_cpu () {
+    cpu_t cpu = {.pc = 0, .sp = -1, .state = Cpu_Running,
+                 .steps = 0, .stack = {0},
+                 .pmem = LoadedProgram ? LoadedProgram : DefProgram};
+    return cpu;
+}
+
+static const char *steplimit_opt = "--steplimit=";
+static const char *inp_prog_opt = "--inp-prog=";
+
+static inline
+void help_dump(char * exec_name, int ret_code) {
```

Comment:
This function should be better named, like `report_usage_and_exit()`, or just `usage_and_exit()`. It is important to make sure its name reflects its non-returning logic (the fact that it calls `exit()`).